ray_on_golem/server/services/yagna.py

```diff
@@ -26,23 +30,18 @@ def __init__(self, yagna_path: Path):
         self._yagna_process: Optional[Process] = None
 
     async def init(self) -> None:
-        self.yagna_appkey = await self._get_or_create_yagna_appkey()
-
         if await self._check_if_yagna_is_running():
             logger.info("Yagna service is already running")
         else:
             await self._run_yagna_service()
+            await self._run_yagna_payment_fund()  # FIXME
```

Comment:
`yagna payment fund` is needed only when yagna is launched the first time - or - when it depletes testnet fund (be it ETH or tGLM)

on a regular restart, only `yagna payment init` is currently required (and, actually, could be done without if we ported the mechanism currently used in yapapi)

Reply:
Changed to `payment init`.